lib/juice.js

```diff
@@ -220,10 +220,16 @@ function inlineDocument($, css, options) {
     var elName = el.name.toUpperCase();
     if (juice.widthElements.indexOf(elName) > -1) {
       for (var i in el.styleProps) {
-        if (el.styleProps[i].prop === 'width' && el.styleProps[i].value.match(/px/)) {
-          var pxWidth = el.styleProps[i].value.replace('px', '');
-          $(el).attr('width', pxWidth);
-          return;
+        if (el.styleProps[i].prop === 'width') {
+          if (el.styleProps[i].value.match(/px/)) {
+            var pxWidth = el.styleProps[i].value.replace('px', '');
+            $(el).attr('width', pxWidth);
+            return;
+          }
+          if (juice.tableElements.indexOf(elName) > -1 && el.styleProps[i].value.match(/\%/)) {
```

Comment:
Oh, yeah, I was thinking about setting image widths in `style` which is obviously not what this does. Do you mind just updating tests to cover this and then it can be merged?